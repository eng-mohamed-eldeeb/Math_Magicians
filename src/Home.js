const Home = () => (
  <div className="wilcom">
    <h1>Wilcome Wilcome to the page</h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio nemo magni,
      nesciunt non doloribus itaque perferendis, expedita sit minus rem
      suscipit, quisquam debitis quis? Voluptates corrupti nostrum, iure numquam
      suscipit pariatur asperiores nemo eaque reprehenderit, earum voluptatibus
      enim. Sint officiis necessitatibus ipsa accusamus earum unde assumenda
      nihil, saepe nam labore dolore molestiae natus consequatur temporibus
      error modi quisquam quaerat perspiciatis
    </p>
  </div>
);
export default Home;
