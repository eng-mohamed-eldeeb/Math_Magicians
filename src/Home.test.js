import React from "react";
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

// import renderer from 'react-test-renderer';
import Home from "./Home";
it('it works', () => {
    const tree = render(
      <Home />)
  expect(tree).toMatchSnapshot();
})

it('renders correctly', () => {
    render(<Home />);
    expect(screen.getByText('Wellcome Wellcome to the page')).toBeInTheDocument();
})