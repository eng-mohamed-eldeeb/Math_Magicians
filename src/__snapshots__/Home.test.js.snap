// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="wilcom"
      >
        <h1>
          Wellcome Wellcome to the page
        </h1>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio nemo magni, nesciunt non doloribus itaque perferendis, expedita sit minus rem suscipit, quisquam debitis quis? Voluptates corrupti nostrum, iure numquam suscipit pariatur asperiores nemo eaque reprehenderit, earum voluptatibus enim. Sint officiis necessitatibus ipsa accusamus earum unde assumenda nihil, saepe nam labore dolore molestiae natus consequatur temporibus error modi quisquam quaerat perspiciatis
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="wilcom"
    >
      <h1>
        Wellcome Wellcome to the page
      </h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio nemo magni, nesciunt non doloribus itaque perferendis, expedita sit minus rem suscipit, quisquam debitis quis? Voluptates corrupti nostrum, iure numquam suscipit pariatur asperiores nemo eaque reprehenderit, earum voluptatibus enim. Sint officiis necessitatibus ipsa accusamus earum unde assumenda nihil, saepe nam labore dolore molestiae natus consequatur temporibus error modi quisquam quaerat perspiciatis
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
